src/components/Hero.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import heroHome from "../assets/images/heroHome.jpg"
+import { StaticImage } from "gatsby-plugin-image"
+import YohDevButton from "../components/reusuables/YohDevButton"
+
+const sectionStyle = {
+  backgroundImage: `url(${heroHome})`,
+}
+
+const Hero = (props) => {
```

Comment:
Destructure props like the last review.
(props) should be ({heading, subtitle, etc})